### DISCUSSION TITLE
Please tell me your favorite movie(好きな映画を教えてください)

### DISCUSSION THREAD

**Discussion by jomi-99810** [NAIST-SE/AttractingContributors#448] [General]
Please tell me your favorite movie!(好きな映画を教えてください！)

I like  'The Big Blue'.

Link : https://github.com/jomi-99810/AttractingContributors_favoriteMovie

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
